lighthouse-core/audits/third-party-summary.js

```diff
@@ -14,25 +14,30 @@ const NetworkRecords = require('../computed/network-records.js');
 const MainThreadTasks = require('../computed/main-thread-tasks.js');
 
 const UIStrings = {
-  /** Title of a Lighthouse audit that identifies the code on the page that the user doesn't control. This is shown in a list of audits that Lighthouse generates. */
-  title: 'Third-Party Usage',
+  /** Title of a diagnostic audit that provides details about the code on a web page that the user doesn't control (referred to as "third-party code"). This descriptive title is shown to users when the amount is acceptable and no user action is required. */
+  title: 'Third-Party usage',
+  /** Title of a diagnostic audit that provides details about the code on a web page that the user doesn't control (referred to as "third-party code"). This imperative title is shown to users when there is a significant amount of third-party execution time that should be reduced. */
+  failureTitle: 'Reduce impact of third-parties',
```

Comment:
"Third Party" here seems ambiguous.  Third party _code_?  I think that the title could be more clear, as well as the description.

Reply:
yeah sg